### DISCUSSION TITLE
Use of MHDoS on termux

### DISCUSSION THREAD

**Discussion by justachillinguy** [MHProDev/MHDDoS#329] [Ideas]
Hello MHDoS developers!
Can you tell me how do I use this on termux (a linux terminal in Android, app), I really wanted to use this code there.
Thank you for your attention

**Comment by A6-MY** [comment]
termux安装时报错

**Comment by tejvx** [comment]
> Hello MHDoS developers!
> Can you tell me how do I use this on termux (a linux terminal in Android, app), I really wanted to use this code there.
> Thank you for your attention

# Install Proot Distro Than Ubuntu In It And  Than Write These Commands:
- proot-distro login ubuntu
- apt-get install python3
- cd /sdcard/Download/MHDDoS-2.4.4
- pip install -r requirements.txt
- python start.py

**Reply by justachillinguy** [reply]
Thank you bro!

**Reply by ta221156ads1245-rgb** [reply]
Can I do it in alpine(proot-distro)instead of ubuntu.My phone has only 2GB of free space.
